tests/e2e/test_api.py

```diff
@@ -54,6 +54,7 @@ def test_invalid_question():
         "conversation_id": conversation_id,
         "response": expected_details,
         "referenced_documents": [],
+        "truncated": False,
```

Comment:
any reason you only did this on one of the e2e tests?

we don't need to test every field in every e2e test, but at least checking for it in a `valid` response flow and not just a rejected one seems to have value.